src/wanderer/guzzlr.ts

```diff
@@ -21,11 +21,12 @@ function considerAbandon(locationSkiplist: Location[]) {
   );
 
   if (
+    Guzzlr.canAbandon() &&
     // consider abandoning
-    !location || // if mafia faled to track the location correctly
-    locationSkiplist.includes(location) ||
-    !canAdventureOrUnlock(location) || // or the zone is marked as "generally cannot adv"
-    (globalOptions.ascend && wandererTurnsAvailableToday(location) < remaningTurns) // or ascending and not enough turns to finish
+    (!location || // if mafia failed to track the location correctly
+      locationSkiplist.includes(location) ||
+      !canAdventureOrUnlock(location) || // or the zone is marked as "generally cannot adv"
```

Comment:
i do not think this is necessarily needed in this PR, but -- regarding the abandonment of quests, i think there is a case to also calculate, via # of turns to complete the quest, how many of the content unlockers the user would have to buy in order to do the premium quest and using it to help solve this one. my recommendation would be to store this value and subtract it in to the wanderer value. canAdventureOrUnlock (in theory?) should make absinthe and transponder zones as plausibly available but the cost of those should be incorporated in garbo's decision of which wanderer source to use.

Reply:
Unfortunately that's much harder than it seems; backups get used densely for the most part but are sometimes used to reset extro. Digitizes are used at set intervals, as are voters and kramco, but the confluence of all 3 makes math hard. Throw in free for all and a few other sources and you've got yourself a nightmare stew.